src/main/kotlin/com/github/pemistahl/lingua/internal/util/extension/MapExtensions.kt

```diff
@@ -19,3 +19,6 @@ package com.github.pemistahl.lingua.internal.util.extension
 internal fun <T> MutableMap<T, Int>.incrementCounter(key: T) {
     this[key] = this.getOrDefault(key, 0) + 1
 }
+internal fun <T> MutableMap<T, Int>.incrementCounter(key: T, wordSize: Int) {
+    this[key] = this.getOrDefault(key, 0) + wordSize
+}
```

Comment:
Two things here:

1. I wouldn't call the new argument `wordSize` but simply `amount`. Because, in the way you implemented it, `wordSize` is not the length of a single word but the number of words in the input string.

2. The former method in lines 19-21 is now obsolete and can be removed.

Reply:
> @Yunin Can you please work on the aspects that I mentioned? You are perfectly right that words for languages supporting logograms are not counted correctly as of yet. However, unfortunately, your approach is not enough in solving this problem and would produce several other bugs. If you put more work into it, I think we will be able to come to a state that can be merged eventually. Thank you! :)

Thanks for your reply. The several moifications you mentioned are right. I will change the code according to your suggestions. Thanks a lot.